ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -294,6 +294,12 @@ tests:
   steps:
     cluster_profile: openstack-vexxhost
     workflow: openshift-e2e-openstack-byon
+- as: e2e-openstack-proxy
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    env:
+      BASTION_FLAVOR: m1.small
```

Comment:
you can remove it, we have it here now: https://github.com/openshift/release/blob/9a797239e19dacf2dda37e2b5e97c4b68cc72b44/ci-operator/step-registry/openstack/conf/resourcenames/openstack-conf-resourcenames-commands.sh#L29-L35